code/index.html

```diff
@@ -10,24 +10,42 @@
       rel="stylesheet" />
     <title>Chat Bot</title>
   </head>
+  
+  <title>Chat Bot</title>
+</head>
 
-  <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
-    <main>
-      <section class="chat" id="chat"></section>
+<body>
+  <h1>Welcome to my world.</h1>
+  <h2>Ask about whatever.</h2>
+  <main>
+    <section class="chat" id="chat"></section>
+    
+<!--These buttons could have been in the JS as innerHTML, would have done if we had more time-->
```

Comment:
Great that you have reflected on this but also creative solution to your problem!